services/movie.py

```diff
@@ -6,6 +7,7 @@
 def get_movies(
     genres_ids: list[int] = None,
     actors_ids: list[int] = None,
+    title: list[str] = None,
```

Comment:
The `title` parameter is annotated as `list[str]`, but it seems to be intended for a single string value. If you want to filter by a single title, consider changing the type annotation to `str` or `Optional[str]` if it can be None.